src/main/kotlin/mynameisjeff/skyblockclientupdater/utils/UpdateChecker.kt

```diff
@@ -133,6 +134,8 @@ object UpdateChecker {
             for (modEntry in latestMods.keys) {
                 val distance = getLevenshteinDistance(fileName, modEntry)
                 if (distance !in 1..6) continue
+                val er = ExpectedAndReceived(fileName, modEntry)
```

Comment:
This check is useless because it only runs after the distance check

Reply:
intentional, it is supposed to check for example:
differences in strings: 
ABC_1.0.jar -> ABC 1.0.jar => no update, changes are only blacklisted chars
ABC_1.0.jar -> ABC_1.1.jar => update, changes are numeric values, new value is higher
ABC_1.1.jar -> ABC_1.0.jar => no update, changes are numeric values, new value is lower